omnibus-base-mainnet.toml

```diff
@@ -0,0 +1,127 @@
+name = "synthetix-omnibus"
+version = "3.3.4"
+description = "Includes the full synthetix system with configurations applied"
+include = [
+    "tomls/settings-mainnet.toml",
+    "tomls/core.toml",
+    "tomls/permissions.toml",
+    "tomls/collaterals/weth.toml",
+    "tomls/pools/spartan-council.toml",
+    "tomls/markets/spot/eth.toml",
+    "tomls/permit-all-transferCrossChain.toml",
+]
+
+[setting.target_preset]
+defaultValue = "main"
+
+[setting.salt]
+defaultValue = "0x3330363734343032330000000000000000000000000000000000000000000000"
+
+[setting.snx_package]
+defaultValue = "synthetix:3.3.4"
+
+[setting.spot_market_package]
+defaultValue = "synthetix-spot-market:3.3.4"
+
+[setting.perps_market_package]
+defaultValue = "synthetix-perps-market:3.3.4"
+
+[setting.owner]
+defaultValue = "0x6E1613B5c68B4Cf2A58400D8019a97849A678139"
+
+#[setting.snx_address]
+#defaultValue = "0x"
+#[setting.snx_aggregator_address]
+#defaultValue = "0x"
+[setting.weth_address]
+# https://docs.base.org/base-contracts/#base-mainnet
+defaultValue = "0x4200000000000000000000000000000000000006"
+
+[setting.eth_aggregator_address]
+# https://docs.chain.link/data-feeds/price-feeds/addresses/?network=base
+defaultValue = "0x71041dddad3595F9CEd3DcCFBe3D1F4b0a16Bb70"
+
+#[setting.snx_liquidation_reward]
+#defaultValue = "10000000000000000000"
+[setting.minimum_liquidity_ratio]
+defaultValue = "2000000000000000000"
+
+[setting.account_timeout_withdraw]
+defaultValue = "86400"
+
+[setting.deployer]
+defaultValue = "0x302d2451d9f47620374B54c521423Bf0403916A2"
```

Comment:
from our first deployment of v3, the deployer has since changed to `0xEde8a407913A874Dd7e3d5B731AFcA135D30375E`

Reply:
apply same fix to all other of the same address